lib/npm.js

```diff
@@ -5,8 +5,16 @@ let {entries: getEntries, keys: getKeys} = Object
 import {componentManifest as skeleton} from "./skeletons.js"
 import settings from "../config/settings.js"
 import mappings from "../config/mappings.js"
-import origamiComponentNames from "../config/components.js"
+import RepoDataClient from "@financial-times/origami-repo-data-client"
 
+const origamiComponentNames = (async function() {
+	const repoData = new RepoDataClient()
+	const components = await repoData.listRepos({
+		type: 'module'
+	})
+	const componentNames = components.map(component => component.name)
+	return componentNames
+}())
 /**
```

Comment:
i really like this ifee memo style, i've seen you do it in a few places

Reply:
I think I like it to :-)